### PR TITLE
create l8n folder and readme

### LINKED ISSUE #129: Create a place for language-specific localization guides
User story: As a project manager and contributor to one of the projects under the CNCF umbrella, I want a set of CNCF localization guides the localization teams on my project can use. Or, I want a set of guides whose licensing permits me to freely I can copy and adapt them to meet my project's needs. 

Currently, it seems like a set of guides that fully meets these requirements might not be available. Various guides are available on the Internet, but lack the licensing information for CNCF projects to copy and re-use them.

### PR DESCRIPTION
**Pull request by rolfedh**
Fixes #129

### CODE REVIEW

**Comment by nate-double-u** [comment]
Thanks for this, @rolfedh!

While I like the idea of organizing these collections into topic folders, the blurb suggests that we bring in published content from elsewhere. We've so far not brought in any out-of-house docs as guides -- I'm happy to have new ones written, but I'm not sure that we should bring in previously published guides as that introduces a maintenance burden (when do we update our guides based on upstream changes). I think linking to these from an index page may be a better solution.

File: docs/localization/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# CNCF Localization Guides 
+
+This directory contains a growing collection of localization guides that CNCF projects can freely use, copy, and adapt.
+
+Every guide here must meet the following requirements:
```
Comment (nate-double-u) [inline docs/localization/README.md:5]: I like that you're setting requirements expectations here -- we may want to adopt this more generally.

**Comment by rolfedh** [comment]
>

**Comment by rolfedh** [comment]
I hear you, Nate. The purpose articulated in the issue #129 includes this:
"Various guides are available on the Internet, but lack the licensing information for CNCF projects to copy and re-use them."

The intention would be to create a library of re-usable resources for CNCF projects, which might save them from duplicated effort and delays. We can talk about this in our upcoming meeting.

**Comment by nate-double-u** [comment]
Hi @rolfedh, sorry for the delay, I just got back from vacation and am catching back up.

I understand your rationale, and think I agree. I’ll merge this as is so as to default to action — we can discuss and make changes as we go. 🙂

**Pull request merged**